cli/tests/pcluster/validators/test_all_validators.py

```diff
@@ -224,14 +224,14 @@ def test_validators_are_called_with_correct_argument(test_datadir, mocker):
         any_order=True,
     )
     key_pair_validator.assert_has_calls([call(key_name="ec2-key-name")])
-    instance_type_validator.assert_has_calls([call(instance_type="t2.micro")])
+    instance_type_validator.assert_has_calls([call(instance_type="c5d.xlarge")])
```

Comment:
It's not clear to me why you are removing the t2.micro case.
I was expecting to add tests/assertions not to replace existing ones?
May you please clarify?

Reply:
This is to use a instance type that has an instance store

Reply:
Got it: so we are replacing t2.micro with c5d.xlarge because the paths triggered by the latter is a superset of the paths triggered by the former